zerver/openapi/zulip.yaml

```diff
@@ -6700,7 +6700,137 @@ paths:
                       - $ref: "#/components/schemas/JsonError"
                       - example: {"msg": "User is not muted", "result": "error"}
                         description: An example JSON response for when the user is not previously muted
+  /users/me/apns_device_token:
+    post:
+      operationId: add-apns-device-token
+      summary: Add APNs device token
+      tags: ["users"]
+      description: |
+        This endpoint adds an APNs device token to register for IOS push notifications.
 
```

Comment:
Should this be `iOS`?

Reply:
Thanks for pointing that out! Fixed it :)